client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -321,5 +312,12 @@ export default connect(
 		errorNotice,
 		infoNotice,
 		removeNotice,
+		trackTryRewind: siteSlug =>
+			withAnalytics(
+				recordTracksEventAction( 'calypso_disconnect_jetpack_try_rewind' ),
+				navigate( `/stats/activity/${ siteSlug }` )
+			),
```

Comment:
It only occurred to me now that we could've done this even easier: By simply adding `/stats/activity/${ siteSlug }` as the `<Button />`'s `href` prop (and only using `trackTryRewind: () => recordTracksEventAction( 'calypso_disconnect_jetpack_try_rewind' )` as its `onClick`). That's how we do it elsewhere, e.g. on this component's 'Disconnect'  button 😄

Reply:
Yeah, that's quite possible. For tracking, I typically don't look at the Redux history so much but just do `localStorage.setItem('debug', 'calypso:analytics:tracks' )` and watch the console.

Reply:
Just tested this and it's great, it's very useful, thanks Bernie!